src/reducer.test.js

```diff
@@ -0,0 +1,72 @@
+import reducer from "./reducer";
+import { changeTodo, addTodo, deleteTodo } from "./actions";
+
+describe("reducer", () => {
+  it('renders initial state', () => {
+    const state = reducer();
+
+    expect(state.newId).not.toBeUndefined();
+  })
+
+  describe("CHANGE_TODO", () => {
+    it("state 'taskTitle' changes", () => {
+      const state = reducer(
+        { newId: 100, tasks: [], taskTitle: "" },
+        changeTodo("4주차 과제하기")
+      );
+
+      expect(state.taskTitle).toBe("4주차 과제하기");
+    });
+  });
```

Comment:
```suggestion
  const initialState =  { newId: 100, tasks: [], taskTitle: "" };
  
  describe("changeTodo", () => {
    it("changes taskTitle", () => {
      const state = reducer(initialState, changeTodo("4주차 과제하기"));

      expect(state.taskTitle).toBe("4주차 과제하기");
    });
  });
```

초기 상태를 변수로 추출하면 더 단순할 것 같아요. 혹은 초기 상태가 비어있다라는 것을 더 강조하고 싶다면 다음과 같이 할 수 있을 것 같아요

```js
  const initialState =  { newId: 100, tasks: [], taskTitle: "" };
  
  describe("changeTodo", () => {
    it("changes taskTitle", () => {
      const state = reducer({ ...initialState, taskTitle: '' }, changeTodo("4주차 과제하기"));

      expect(state.taskTitle).toBe("4주차 과제하기");
    });
  });
```

Reply:
아래 써주신 코드가 정말 깔끔하네요 명확한 코드에 대해서 배웠습니다